ricecooker/commands.py

```diff
@@ -1,64 +1,174 @@
 import os
+import sys
+import requests
+import json
 import webbrowser
-from ricecooker.config import STORAGE_DIRECTORY
+from ricecooker import config
 from ricecooker.classes import nodes, questions
-from ricecooker.managers import ChannelManager
+from requests.exceptions import HTTPError
+from ricecooker.managers import ChannelManager, RestoreManager, Status
+
+def uploadchannel(path, debug, verbose=False, update=False, resume=False, reset=False, step=Status.LAST.name, token="#", **kwargs):
 
-def uploadchannel(path, domain, verbose=False, **kwargs):
     """ uploadchannel: Upload channel to Kolibri Studio server
         Args:
             path (str): path to file containing channel data
-            domain (str): domain to upload to
+            debug (bool): determine which domain to upload to
             verbose (bool): indicates whether to print process (optional)
         Returns: (str) link to access newly created channel
     """
+    # Get domain to upload to
+    domain = config.PRODUCTION_DOMAIN
+    if debug:
+      domain = config.DEBUG_DOMAIN
```

Comment:
Plus do we do anything else special with `--debug`, aside from setting `DEBUG_DOMAIN`? If not, let's remove it and just use `CONTENTWORKSHOP_URL`.